site-packages/integralstor/update_system_date_time.py

```diff
@@ -0,0 +1,28 @@
+from integralstor_utils import command
+
+def update_system_date_time(date, time):
+    try:
+	if not time == None:
+            cmd = 'date -s %s' %time
+            output, err = command.get_command_output(cmd)
+            if err:
+                raise Exception(err)
+        if not date == None:
+            date = date.split("/")
+            date = date[2] + "-" + date[0] + "-" + date[1]
```

Comment:
How can you be sure that the split always returns 3 parts? What will your code do it it returns 2 parts? It will throw an exception.l